### DISCUSSION TITLE
backend start failure: <nil>

### DISCUSSION THREAD

**Discussion by Pulasthi91** [utmstack/UTMStack#835] [Q&A] [answered]
After 4 hour long installation i got  **backend start failure: <nil>**. I did clean installation by providing suggested system requirement. Any solution to troubleshoot **backend start failure: <nil>** issue, without installing from the scratch.

**Comment by c3s4rfred** [comment, marked as answer]
Hi, @Pulasthi91, consider to run the installer again and tell us what happen. Backend needs the postgres database container and opensearch container to be ready and have connection to them before start, if you have some network issues, backend fails.

Best regards
